src/main/scala/firrtl/PrimOps.scala

```diff
@@ -102,7 +102,7 @@ object PrimOps extends LazyLogging {
   private lazy val builtinPrimOps: Seq[PrimOp] =
     Seq(Add, Sub, Mul, Div, Rem, Lt, Leq, Gt, Geq, Eq, Neq, Pad, AsUInt, AsSInt, AsClock, Shl, Shr,
         Dshl, Dshr, Neg, Cvt, Not, And, Or, Xor, Andr, Orr, Xorr, Cat, Bits, Head, Tail)
-  private lazy val strToPrimOp: Map[String, PrimOp] = builtinPrimOps map (op => op.toString -> op) toMap
+  private lazy val strToPrimOp: Map[String, PrimOp] = builtinPrimOps.map { case op : PrimOp=> op.toString -> op }.toMap
 
```

Comment:
I had a hard time converting this statement.  my ide preferred using case.  I am not a fan of yield, it's clear enough if you really understand for comprehensions, but I find the asymmetry with ordinary usages of for such as the inability to do intermediate calculations before the yielded quantity and the difficulties in inserting breakpoints in the for comprehensions sufficient to keep me from it.